sdk/eventhub/event-hubs/src/eventPosition.ts

```diff
@@ -226,3 +83,76 @@ export function isLatestPosition(eventPosition: EventPosition): boolean {
 
   return false;
 }
+
+/**
+ * Gets the `EventPosition` corresponding to the location of the the first event present in the partition.
+ * Pass this position to the `EventHubConsumerClient.subscribe()` method to begin receiving events from the
+ * first event in the partition which has not expired due to the retention policy.
+ */
+export const earliestEventPosition: EventPosition = {
+  offset: -1
+};
+
+/**
+ * Gets the `EventPosition` corresponding to the end of the partition.
+ * Pass this position to the `EventHubConsumerClient.subscribe()` method to begin receiving events from the
+ * event that is enqueued right after the method call.
+ * @returns EventPosition
+ */
+export const latestEventPosition: EventPosition = {
+  offset: "@latest"
+};
+
+/**
+ * @ignore
+ * @internal
+ */
+export function validateEventPositions(
+  position: EventPosition | { [partitionId: string]: EventPosition }
+) {
+  if (position == undefined) {
+    return;
+  }
+
+  const offsetPresent = position.offset != undefined;
+  const sequenceNumberPresent = position.sequenceNumber != undefined;
+  const enqueuedOnPresent = position.enqueuedOn != undefined;
+  const keys = Object.keys(position);
+
+  if (offsetPresent || sequenceNumberPresent || enqueuedOnPresent || !keys.length) {
```

Comment:
For the `!keys.length` condition, would it be helpful to create a different error message that indicates that the user can pass in a map of partitionIds to EventPosition? If we're trying to promote using a single EventPosition instead of a map then I think the current error message is fine.

Reply:
I tried this. The error message became way too long. I say, let's keep the current state for now and change based on feedback later